test/saithrift/tests/switch.py

```diff
@@ -120,7 +120,7 @@ def sai_thrift_flush_fdb_by_vlan(client, vlan_id):
     fdb_attr_list = [fdb_attribute1, fdb_attribute2]
     client.sai_thrift_flush_fdb_entries(thrift_attr_list=fdb_attr_list)
 
-def sai_thrift_create_virtual_router(client, v4_enabled, v6_enabled):
+def sai_thrift_create_virtual_router(client, v4_enabled, v6_ienabled,mac):
```

Comment:
rename v6_ienabled to v6_enabled. can you also add space after the comma to keep consistent code style